src/main/java/org/spongepowered/common/data/SpongeDataRegistry.java

```diff
@@ -206,6 +202,11 @@ public static void finalizeRegistration() {
         }
         checkArgument(!immutableProcessorList.contains(processor), "Duplicate DataProcessor Registration!");
         immutableProcessorList.add(processor);
+        
+        SpongeSerializationService service = (SpongeSerializationService) Sponge.getGame().getServiceManager().provide(SerializationService.class).get();
```

Comment:
This is also a no no, because it really really prevents any possibility of running `ManipulatorTest` since using anything with `Sponge` will result in complete failure.